build.sbt

```diff
@@ -8,41 +9,18 @@ name := "facia-api-client"
 description := "Scala client for The Guardian's Facia JSON API"
 
 val sonatypeReleaseSettings = Seq(
-  licenses := Seq("Apache V2" -> url("http://www.apache.org/licenses/LICENSE-2.0.html")),
-  scmInfo := Some(ScmInfo(
-    url("https://github.com/guardian/facia-scala-client"),
-    "scm:git:git@github.com:guardian/facia-scala-client.git"
-  )),
-  pomExtra := (
-    <url>https://github.com/guardian/facia-scala-client</url>
-      <developers>
-        <developer>
-          <id>janua</id>
-          <name>Francis Carr</name>
-          <url>https://github.com/janua</url>
-        </developer>
-        <developer>
-          <id>adamnfish</id>
-          <name>Adam Fisher</name>
-          <url>https://github.com/adamnfish</url>
-        </developer>
-      </developers>
-    ),
```

Comment:
All the [required metadata](https://central.sonatype.org/publish/requirements/#project-name-description-and-url) deleted here is now provided by [`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow) ([quite concisely](https://github.com/guardian/gha-scala-library-release-workflow/blob/d6ea7cc3eff7868b5ec59879ff0ebf8c044dd8cb/.github/workflows/reusable-release.yml#L158), thanks to `sbt-sonatype`'s `GitHubHosting` helper!).